src/libs/actions/Policy.js

```diff
@@ -662,34 +669,75 @@ function updateWorkspaceCustomUnit(policyID, currentCustomUnit, values) {
 
 /**
  * @param {String} policyID
+ * @param {Object} currentCustomUnitRate
  * @param {String} customUnitID
  * @param {Object} values
  */
-function setCustomUnitRate(policyID, customUnitID, values) {
-    DeprecatedAPI.Policy_CustomUnitRate_Update({
-        policyID: policyID.toString(),
-        customUnitID: customUnitID.toString(),
-        customUnitRate: JSON.stringify(values),
-        lastModified: null,
-    })
-        .then((response) => {
-            if (response.jsonCode !== 200) {
-                throw new Error();
-            }
+function updateCustomUnitRate(policyID, currentCustomUnitRate, customUnitID, values) {
```

Comment:
NAB: rename values to newCustomUnitRate.

It took me a while to understand the difference between `currentCustomUnitRate` and `values` because when i looked at a snapshot they looked the same. Then i noticed just values were different. 
I think we could avoid this confusing for others by renaming `values` to be more verbose i.e., `newCustomUnitRate`.